_episodes/02-make-it-public.md

```diff
@@ -265,5 +265,53 @@ Main points for this section:
 
 ## Making your software reusable with Docker
 
-## What if your software is an analysis in a Notebook?
+### What's Docker
+
+[Docker](https://www.docker.com) is an open source project that automates the deployment of software applications in a sandbox (called containers).
+
+> Containers are a way to package software in a format that can run isolated on a operating system, e.g. Linux. Unlike virtual machines, containers do not bundle a full operating system - only libraries and settings, required to make the software work. This makes containers efficient, lightweight, self-contained systems and guarantees that software runs reproducibly, regardless of where it is deployed.
```

Comment:
```suggestion
> Containers are a way to package software so that it can be run isolated on an operating system, e.g. Linux. Unlike virtual machines, containers do not bundle a full operating system - only libraries and settings, required to make the software work. This makes containers efficient, lightweight systems and guarantees that software runs reproducibly, regardless of where it is deployed.
```